airflow/airflow_test.go

```diff
@@ -85,3 +93,37 @@ func TestInit(t *testing.T) {
 		assert.True(t, exist)
 	}
 }
+
+func Test_airflowVersionFromDockerFile(t *testing.T) {
+	airflowHome := config.WorkingPath + "/testfiles"
+
+	// Version 1
+	expected := uint64(0x1)
```

Comment:
shouldnt this be different functions or a table test?

Reply:
I had just copy pasted these tests, as I have moved the original function, but will move it to table test 👍

Reply:
updated

Reply:
ty